index.html

```diff
@@ -230,5 +230,53 @@ <h2 class="yodaa">About Myself</h2>
         </div>
       </div>
     </section>
+    <section class="form">
+      <form
+        method="post"
+        action="https://formspree.io/f/{form_id}"
+        class="form-section"
```

Comment:
- [ ] Please as advised by the previous reviewer, You need to generate an id for your form so that the form can submit its content. Read this resource to [setup formspree](https://formspree.io/html/) 😄